libraries/api-documenter/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-documenter",
+  "version": "0.1.0",
```

Comment:
Fixed, per our discussion.

---
In reply to: [134042716](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134042716) [](ancestors = 134042716,134040990)